envergo/analytics/middleware.py

```diff
@@ -1,9 +1,10 @@
 from uuid import uuid4
 
-from django.conf import settings
-
 from config.settings.base import VISITOR_COOKIE_NAME
```

Comment:
Je ne sais pas pourquoi j'ai fait ça comme ça à la base, mais on ne devrait jamais importer directement depuis les modules de settings. La bonne est syntaxe est bien d'importer `from django.conf import settings`. Sinon on s'expose à divers problèmes. Par exemple, si on déplace cette valeur, le code devient invalide.

https://docs.djangoproject.com/en/dev/topics/settings/#using-settings-in-python-code

> Also note that your code should not import from either global_settings or your own settings file. django.conf.settings abstracts the concepts of default settings and site-specific settings; it presents a single interface. It also decouples the code that uses settings from the location of your settings.